### DISCUSSION TITLE
I would like to play audio in my flashcard app

### DISCUSSION THREAD

**Discussion by GirkovArpa** [c-smile/sciter-js-sdk#42] [Q&A] [answered]
I made [this](https://github.com/GirkovArpa/slidercards) little flashcard app and would like to play a ding whenever the user gets the answer correct, but searching this repo for the keywords "sound" and "audio" did not lead me to any examples of how to do so.  Perhaps it is not possible yet, but I am crossing my fingers 🤞

**Comment by c-smile** [comment, marked as answer]
In the next build you will be able to play audio, like this:

```JavaScript 
        document.on("click", "button#play", async function(evt,btn) {
            btn.state.disabled = true;
            var audio = await Audio.load(__DIR__ + "/hasta-lavista-baby.mp3");
            await audio.play();
            btn.state.disabled = false;
        });
```
